tee-worker/litentry/core/identity-verification/src/web2/mod.rs

```diff
@@ -20,13 +20,14 @@ extern crate sgx_tstd as std;
 #[cfg(all(feature = "std", feature = "sgx"))]
 compile_error!("feature \"std\" and feature \"sgx\" cannot be enabled at the same time");
 
+mod discord;
```

Comment:
hmm discord is private mod while twitter is public mod, why?

Reply:
@Kailai-Wang  does it make sense to clean up a bit the "pub" key words in our code in general ? I think there are quite some places where ”pub" is not necessary.

Reply:
> hmm discord is private mod while twitter is public mod, why?
Discord's oauth mechanism is a bit different, there is no need to generate and store a code challenges. This is why there are no exports for discord

Reply:
I see, thanks - maybe we could have `mod twitter;` and only export the symbols that we need, but it's a minor problem (maybe there're too many symbols to export, which can be very verbose)